setup.py

```diff
@@ -10,64 +10,66 @@
 # warranty.
 #
 
-from setuptools import setup, find_packages
 import subprocess
 
+from setuptools import find_packages, setup
+
 
 def get_git_revision_hash():
     try:
-        git_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD'])
+        git_hash = subprocess.check_output(["git", "rev-parse", "HEAD"])
     except subprocess.CalledProcessError:
-        return 'master'
+        return "master"
     else:
-        return git_hash.decode('ascii').splitlines()[0]
+        return git_hash.decode("ascii").splitlines()[0]
 
 
-README = 'https://github.com/torchbox/wagtail-markdown/blob/{hash}/README.md'
-README = README.format(
-    hash=get_git_revision_hash()
-)
+README = "https://github.com/torchbox/wagtail-markdown/blob/{hash}/README.md"
+README = README.format(hash=get_git_revision_hash())
 
 
 INSTALL_REQUIRES = [
-    'Markdown>=3,<4',
-    'bleach>=1.4.2,<2.2',
-    'Wagtail>=2.0',
+    "Markdown>=3,<4",
+    "bleach>=1.4.2,<2.2",
+    "Wagtail>=2.0",
 ]
 
 
 TESTING_REQUIRES = [
-    'dj_database_url==0.4.2',
+    "dj_database_url==0.4.2",
 ]
 
 
 CLASSIFIERS = [
-    'Development Status :: 5 - Production/Stable',
-    'Environment :: Web Environment',
-    'Intended Audience :: Developers',
-    'Operating System :: OS Independent',
-    'Programming Language :: Python',
-    'Framework :: Django',
-    'License :: OSI Approved :: zlib/libpng License',
-    'Programming Language :: Python :: 3',
-    'Framework :: Wagtail',
-    'Framework :: Wagtail :: 2'
+    "Development Status :: 5 - Production/Stable",
```

Comment:
I find it a bit odd this is marked as production/stable and at the same time released as a `0.x` version. According to SemVer `0.x` is for "initial development", with no stability commitments. If this is stable, we should make this `v7.0.0`.

Reply:
Somehow I missed this one. Moved it to beta